portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,25 +23,42 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+// @WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
   @Override
```

Comment:
This should be commented out, right? I think this is accidentally applied to `doPost()`, below all the comments. It'd be better to have `@Override` directly above `doPost()`.

Reply:
Okay, I'll fix that!